### ISSUE TITLE
Looking for Nook DeDRM help

### ISSUE THREAD

**Issue by baqlias** [apprenticeharper/DeDRM_tools#1931]
I haven't bought Nook books for a while, but now I see a Nook book for much less than the Kindle version. I'd like to be able to DeDRM any Nook book I buy.  I have a Calibre setup on my Mac.  The DeDRM setup works for Kindle, but I do not have my old B & N key.  I entered info in the DeDRM tools section for Nook to see if I could generate a working key.  It did not work.  Perhaps things have changed on the Nook end and this cannot be done anymore?  

Any help will be appreciated!  Thanks!

**Comment by Leseratte10** [comment]
The builtin key retrieval doesn't work anymore since the server that's used for this has been shut down. I believe there's a way to set up an Android emulator, install the Nook app, and extract the necessary key from the emulator somehow. 

I did find [these instructions](https://www.aricrenzo.com/2019-12-13-Liberate-Your-Nook-Ebooks/) but I'm not sure if these are the most recent ones or if stuff has changed already.

**Comment by ElleKayEm** [comment]
Yes, I've seen reports in just the last week or so that the method which @Leseratte10 links to still works.  There is a long discussion about it in issue #814.  Some of the more recent comments there might be helpful.

**Comment by baqlias** [comment]
Thanks for your comments.  I've been trying the method but thus far haven't been able to get the Nook app to install in my virtual device.  I'll keep at it when I can!
